libs/ui/src/components/TooltipTab/TooltipTab.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <o-tab-item
+    :value="value"
+    class="py-5 tooltip-tab"
+    :disabled="disabled"
+    :label="label">
+    <template #header>
+      <b-tooltip
```

Comment:
could you please use [oruga-ui tooltip](https://oruga.io/components/tooltip.html) to not mix framework

Reply:
can't we seperate this two component to have something more atomic?

Reply:
you mean a component who is just the tooltip with some styles applies?

Reply:
on one side the `o-tab-item` wrapper with template slot and on the other side the `o-tooltip` wrapper.
Since we'll need to reuse tooltip component.

Reply:
ok. ill seperate them
but first need to figure out why toolyip doesn't show on hover in the app but does show in histoire

Reply:
> ok. ill seperate them but first need to figure out why tooltip doesn't show on hover in the app but does show in histoire

might be some css issue on oruga import? I had a similar problem where I didn't have the same rendering with the app

Reply:
i'm playing around with that but with no result /conclusion
can you provide more direction please?

Reply:
ok i'm closing in on it
the issue comes from somewhere in the themes scss files

Reply:
I am also trying to find a solution, I can see that it works on histoire.
We can try to inspect histoire element & GalleryItem tab to compare the difference.

Reply:
found it

```scss
  body,
  div#__nuxt,
  div#__layout,
  main {
    margin: 0;
    width: 100%;
    height: 100%;
    background-color: $background-dark;
// this line here
    // z-index: 1;
  }
```

mmm should i just delete it? it must have been put there for a reason

Reply:
oh my... nice find! yeah I would remove it it has been introduced in https://github.com/kodadot/nft-gallery/pull/4440 and I think has nothing to do there